flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -4873,6 +4874,12 @@ class FMCMainDisplay extends BaseAirliners {
         return SimVar.GetSimVarValue("CG PERCENT", "Percent over 100") * 100;
     }
 
+    getLandWeight() {
+        this.tryFuelPlanning();
```

Comment:
Hello. I think this causes the bug on the INIT B page with the BLOCK fuel entry logic.

Perhaps can call `this.tryUpdateRouteTrip()` instead of `this.tryFuelPlanning()`